### DISCUSSION TITLE
Crash Reporter

### DISCUSSION THREAD

**Discussion by shuvashish76** [celzero/rethink-app#483] [Ideas]
Can we add our own custom crash reporter as mentioned by AppManager (AM) dev here: https://github.com/MuntashirAkon/AppManager/issues/289#issuecomment-782571555

AM consider [ACRA](https://www.appbrain.com/stats/libraries/details/acra/acra) as Tracker since it phone home by default.

**Comment by ignoramous** [comment]
Any in-app crash reporter wouldn't work for native crashes, detecting OOMs (out-of-memory), or battery kills.

Android 12+ have an API for that, which is what we intend to use: #336
